frontend/src/components/Collection/components/CollectionDatasetsGrid/components/Row/DownloadDataset/index.tsx

```diff
@@ -21,18 +17,17 @@ interface Props {
  * demand.
  */
 const DownloadDataset: FC<Props> = ({
-  Button = StyledButton,
+  Button,
   datasetId,
   isDisabled = false,
   name,
 }) => {
-  // Open state of download modal.
-  const [isOpen, setIsOpen] = React.useState(false);
+  const [isOpen, setIsOpen] = useState<boolean>(false);
 
-  // Function toggling open state of modal.
-  const toggleOpen = useCallback(() => {
-    setIsOpen(!isOpen);
-  }, [isOpen]);
+  // Close modal.
+  const closeModal = () => {
```

Comment:
should we just inline this in ` onClose={() => setIsOpen(false)}` to be consistent with open 😄 ?